base/deprecated.jl

```diff
@@ -3806,6 +3806,32 @@ end
     @deprecate getq(F::Factorization) F.Q
 end
 
+@deprecate method_exists(f, t, world) method_exists(f, t, world = world)
+@deprecate Timer(timeout, repeat) Timer(timeout, repeat = repeat)
+@deprecate Timer(callback, delay, repeat) Time(callback, delay, repeat = repeat)
+@deprecate names(m, all) names(m, all = all)
+@deprecate names(m, all, imported) names(m, all = all, imported = imported)
+@deprecate code_native(io, f, types, syntax) code_native(io, f, types, syntax = syntax)
+@deprecate code_native(f, types, syntax) code_native(f, types, syntax = syntax)
+@deprecate eachmatch(re, str, overlap) eachmatch(re, str, overlap = overlap)
+@deprecate matchall(re, str, overlap) matchall(re, str, overlap = overlap)
+@deprecate chop(s, head) chop(s, head = head)
+@deprecate chop(s, head, tail) chop(s, head = head, tail = tail)
+@deprecate tryparse(T, s, base) tryparse(T, s, base = base)
+@deprecate parse(T, s, base) parse(T, s, base = base)
```

Comment:
Should these have `T::Type{<:Integer}`or something like that?

Reply:
Sure, that sounds right